packages/content/parsers/markdown/index.js

```diff
@@ -70,7 +71,7 @@ class Markdown {
     }
 
     return new Promise((resolve, reject) => {
-      let stream = unified().use(parse)
+      let stream = unified().use(parse).use(gfm)
```

Comment:
The `remark-gfm` plugin should be added here https://github.com/nuxt/content/blob/dev/packages/content/lib/utils.js#L14 to not break options override.